### PR TITLE
Marks up FAQ to support structured data

### PR DESCRIPTION
**Pull request by bshyong**
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4534

Adds support for FAQ structured data to the FAQ page.

Resource for schema validation https://chrome.google.com/webstore/detail/structured-data-testing-t/kfdjeigpgagildmolfanniafmplnplpl

### CODE REVIEW

File: content/pages/faq.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ permalink: /faq.html
 title: Frequently Asked Questions
 display_title: Frequently Asked Questions
 ---
-<main>
+<main itemscope itemtype="http://schema.org/FAQPage">
```
Comment (robbiethegeek-usds) [inline content/pages/faq.md]: around the content describing the FAQPage (after line 11 before line 16) you need : 

<div itemprop="description"> and that closed before the .feature div on line 18

**Review by robbiethegeek-usds — CHANGES_REQUESTED** [review]
If you look on the structured data tool it isn't picking up the FAQPage (even as an error without it).

File: content/pages/faq.md (hunk 1 above)
Comment (bshyong) [reply content/pages/faq.md]: Got it—thanks for catching this! It's showing up in the tool now

**Review by U-DON — COMMENTED** [review]
I think it looks good... Though, when I ran the latest through the tool, it says there's one error on FAQPage.

**Review by robbiethegeek-usds — APPROVED** [review]

**Comment by robbiethegeek-usds** [comment]
Thanks looks good!

**Comment by robbiethegeek-usds** [comment]
FAQPage is a proposed schema, known issue @U-DON but thanks for calling out! :)

**Pull request merged**